meetings/2022-09-22_ElixirConfLunch.md

```diff
@@ -0,0 +1,102 @@
+Attendees:
+- Eric Rauer
+- Ben Wheatley
+- Chris Ertel
+- Jason Johnson
+- Alex McClain
+- Ed Bond
+- Jason Axelson
+
+Location: ElixirConf 2022 Lunch
+
+Governance
+- Are we aligned with Scenic's general direction?
+- Do we want to support our own driver?
+  - Ideally not
+
+JasonA put forth some short term goals:
+- Release 0.11
+- Move scenic's code from boydm/scenic to the ScenicFramework organization
+- Create guides to get started with Scenic
+
+Ideas:
+- Split Scenic into layers
+  - Boyd owns the lower levels
+  - community owns upper levels
+
+Discussion about why do people want to use Scenic
+- Scenic is good for the embedded use case
+- "Acceptable for 2d use cases"
+- Decent choice for UIs?
+- Cooler to not use browser
+- Ed wants to use it on desktop, but not production use case
+- Provides fast/good feedback loop for Desktop
+
+Scenic will always support embedded
+Scenic will not have native OS chrome, we should accept that
+
+Ideas:
+- Tailwind UI like component library for Scenic?
+- Community livebook for Scenic
+- hype-up video like what Lars made for Nerves
+- Try toucan for pairing?
+
+What we want:
+- Jason Johnson
+  - Layouts and better font support
+  - polishing and approachability
+    - We should do a LiveBook
+    - fly.io hosted livebook
+- Alex McClain
+  - Be able to create a UI that doesn't suck (IOT use case)
+    - IOT UIs universally suck
+  - Hardware availability for Scenic
+- Ed Bond
+  - Developer experience for getting started
+  - cross-compiling drivers
+    - here are common things that will go wrong
```

Comment:
```suggestion
    - here are common things that will go wrong
      - target cross compile chain requires deeper knowledge
      - When you [build](https://github.com/boydm/scenic_driver_glfw) with a target OS architecture, which drivers are needed where and why. 
      - When you have that driver as a nested depedency, it can't easily build for targets like RISC processors 
```